vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -3,7 +3,7 @@ def up
     create_table :container_nodes do |t|
       t.string    :ems_ref
       t.string    :name
-      t.timestamp  :creation_timestamp
+      t.timestamp  :creation_timestamp, :null => true
```

Comment:
> It's only for timestamps

Ah, that makes sense then.  All timestamps would be really annoying.  Even so, we need to coordinate our migrations around upgrades as per my other comment.